functional-tests/support/action/windowIsScrolledToBottom.ts

```diff
@@ -1,5 +1,7 @@
 export async function windowIsScrolledToBottom(): Promise<boolean> {
 	return browser.execute(
-		() => window.innerHeight + window.pageYOffset >= document.body.offsetHeight
+		() =>
+			Math.ceil(window.innerHeight + window.pageYOffset) >=
```

Comment:
Noticed that this occasionally didn't return correctly as the window.innerHeight would be xxx.04 where page document.body.offsetHeight would be xxx.06.  So rounding up will make sure it returns correctly.